File: PodBookingSystem.B.ServiceLayer/DTO/Request/CreateAccountRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PodBookingSystem.B.ServiceLayer.DTO.Request
{
    public class CreateAccountRequest
    {
        public string Email { get; set; }
        public string Name { get; set; }
        public string Phone { get; set; }
    }
}

File: app_backup/src/main/java/com/example/g6podbookingsystem/dto/CreateAccountRequest.java
package com.example.g6podbookingsystem.dto;

public class CreateAccountRequest {
    private String email;
    private String name;
    private String phone;

    public CreateAccountRequest(String email, String name, String phone) {
        this.email = email;
        this.name = name;
        this.phone = phone;
    }
}
